torch/csrc/PyInterpreter.cpp

```diff
@@ -12,6 +12,8 @@ using namespace torch;
 using namespace at;
 using namespace c10;
 
+namespace torch::csrc {
```

Comment:
csrc wasn't meant to be a namespace name, maybe torch::detail?

Reply:
Yes I wasn't sure so this was a placeholder.
I'm happy with torch::detail!